### DISCUSSION TITLE
Possible to have on screen elements output values to midi in port?

### DISCUSSION THREAD

**Discussion by CalenShirfield** [djipco/webmidi#420] [Q&A] [answered]
I want to create a web based midi controller that can send cc values to a device. Currently, I have a slider that updates it’s position when receiving rawValue from addlistener(‘controlchange’).
 I also have an addEventListener(‘input’) on slider. I want my device to receive this slider value as a cc event on input.

**Comment by djipco** [comment, marked as answer]
To do that, you can simply use the [`sendControlChange()`](https://webmidijs.org/api/classes/OutputChannel#sendControlChange) method of either the [`Output`](https://webmidijs.org/api/classes/OutputChannel#sendControlChange) or the [`OutputChannel`](https://webmidijs.org/api/classes/OutputChannel) objects. Did you check the documentation? It has an example that's pretty close to what you want to do in the [Sending Outgoing MIDI Messages](https://webmidijs.org/docs/getting-started/basics#sending-outgoing-midi-messages) section.

**Reply by CalenShirfield** [reply]
Thanks djipco.I have hooked up Snoize midi monitor and In order to see the cc's there is an option to spy on output to destinations. I can see they are arriving with the slider value adjustments. However with a launchpad device there are further requirements to get LED feedback. This threw me. Thanks again.
